code_samples/sql_json.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const { Client, SqlColumnType, HazelcastJsonValue } = require('hazelcast-client');
+
+(async () => {
+    try {
+        const client = await Client.newHazelcastClient({
+            properties: {
+                'hazelcast.logging.level': 'OFF'
+            }
+        });
+        const mapName = 'jsonMap' + Math.floor(Math.random() * 10000);
+
+        // To be able to use our map in SQL we need to create mapping for it.
```

Comment:
How about "In order to use the map in SQL a mapping should be created"?

Reply:
Applying..